definitions/features/salt_server.rb

```diff
@@ -0,0 +1,22 @@
+class Features::SaltServer < ForemanMaintain::Feature
+  metadata do
+    label :salt_server
+
+    confine do
+      find_package('salt-master') &&
+        ForemanMaintain::Utils::Service::Systemd.new('salt-master', 0).exist? &&
+        ForemanMaintain::Utils::Service::Systemd.new('salt-master', 0).enabled?
+    end
+  end
+
+  def services
+    salt_services = []
+    salt_services += [system_service('salt-master', 30)]
+    salt_services += if ForemanMaintain::Utils::Service::Systemd.new('salt-api', 0).exist?
+                       [system_service('salt-api', 30)]
+                     else
+                       []
+                     end
```

Comment:
I would have written this as
```suggestion
    salt_services = [system_service('salt-master', 30)]
    if ForemanMaintain::Utils::Service::Systemd.new('salt-api', 0).exist?
      salt_services += [system_service('salt-api', 30)]
    end
```

But wouldn't block on syntax either way